ansible_base/lib/utils/models.py

```diff
@@ -102,6 +103,15 @@ def current_user_or_system_user():
     return user
 
 
+def is_encrypted_field(model, field_name):
+    from django.contrib.auth.models import AbstractUser
+
+    if isinstance(model, AbstractUser) and field_name == 'password':
+        return True
+
+    return field_name in getattr(model, 'encrypted_fields', [])
```

Comment:
What if a field was decorated with `prevent_search`?

https://github.com/ansible/django-ansible-base/blob/392ea7713b4b7b61191982127ed17dd31968f6e8/ansible_base/lib/utils/models.py#L54-L68

Reply:
Sure, added a check for that.